.github/workflows/buildtrackerjs.yml

```diff
@@ -1,8 +1,10 @@
 name: Build Tracker JS files
 
 on:
-  issue_comment:
-    types: [created]
+  push:
+    paths:
+      - 'js/piwik.js'
```

Comment:
The reason I didn't add that in that way in the first place was that it might cause trouble when your branch gets silently updated by a github action without any notice. It might be kind of fine, when that happens for a pull request but I wouldn't do that for a random branch when something is pushed.
Also I'm not sure if we should run such actions on our "protected" branches, like 4.x-dev or the release branches. We wouldn't want to have any automatic commits there.